packages/apollo-server-fastify/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
```

Comment:
This `tsconfig.json` should `extends` the root `tsconfig.json`, in a similar fashion as seen [here](https://github.com/apollographql/apollo-server/blob/c5d0c43b555cb0f6627eaf1ce4dc113935ec4fe1/packages/apollo-server-express/tsconfig.json#L2).

Reply:
There is an existing problem at times with extending the `tsconfig` with fastify, wherein the version of `@types/node` starts to conflict. I opened an [issue](https://github.com/fastify/fastify/issues/908) in fastify for this, but it was very flaky in behaviour on my machine. Will try to use the `extend` again and check